frontend/app/services/docs.js

```diff
@@ -0,0 +1,31 @@
+import Service, { service } from "@ember/service";
+
+export default class DocsService extends Service {
+  timedDocsURL = "https://timed.dev/docs/";
+  @service router;
+
+  docsUrlMatch = {
+    // index.name: DocsUrl
+    "index.attendances": "tracking/attendances",
+    "index.reports": "tracking/timesheet",
+    "analysis.index": "analysis",
+    statistics: "statistics",
+    projects: "projects",
+    "users.index": "users",
+    "users.edit.index": "users",
+    "index.activities.index": "tracking/activities",
+  };
```

Comment:
more of a nitpick but personally i'd prefer this over having it in the service

```suggestion

const DOCS_URL_MAP = {
    // index.name: DocsUrl
    "index.attendances": "tracking/attendances",
    "index.reports": "tracking/timesheet",
    "analysis.index": "analysis",
    statistics: "statistics",
    projects: "projects",
    "users.index": "users",
    "users.edit.index": "users",
    "index.activities.index": "tracking/activities",
  };

export default class DocsService extends Service {
  timedDocsURL = "https://timed.dev/docs/";
  @service router;
  
```